app/controllers/api/v1/ndcs_controller.rb

```diff
@@ -205,8 +211,11 @@ def filtered_indicators(source=nil)
         indicators = indicators.where(source_id: source.map(&:id)) if source
 
         if @indc_locations_documents
+          # TODO: do not understand why below are needed
+          # if indicator belongs to many cateogires then only one will be in category_ids
+          # that's why I'm going to reset the query at the end
           indicators = indicators.select('DISTINCT ON(COALESCE("normalized_slug", indc_indicators.slug)) indc_indicators.*')
```

Comment:
@Bluesmile82 do you know what is normalized_slug and why it is used when passing locations_documents parameter?